package.json

```diff
@@ -9,8 +9,8 @@
   "shortcutFolderName": "MongoDB",
   "license": "SSPL",
   "scripts": {
-    "bootstrap": "npm install && lerna run prepare --stream",
-    "bootstrap-evergreen": "npm ci && lerna run prepare",
+    "bootstrap": "npm install && lerna run prepublishOnly --stream",
```

Comment:
Would it be possible to be less implicit here and really just add a `bootstrap` task to all the packages with that being what we would expect to be called as part of the bootstrap? 

I feel it would be better to avoid overloading `prepublishOnly`, like that we would probably avoid having unrelated tasks being run incidentally in future.

For example we could have probably avoided re-downloading the fonts in the plugins on `prepublishOnly` since those are only used in development.

Reply:
Yeah, let's split them up. `prepare` that was used there before was also running on install and before publish, so this PR just doesn't change the behavior, but your suggestion makes a lot of sense

Reply:
Done in 7c071d7, wanna give the PR another look?